clientconn.go

```diff
@@ -435,6 +464,24 @@ func (cc *ClientConn) lbWatcher() {
 	}
 }
 
+func (cc *ClientConn) scWatcher() {
+	for {
+		select {
+		case sc, ok := <-cc.dopts.sc:
+			if !ok {
+				return
+			}
+			cc.mu.Lock()
+			// TODO: load balance policy runtime change is ignored.
```

Comment:
We should definitely fix this (although it doesn't need to be part of this PR).  We need to support the case where (e.g.) a server name switches from pick_first to grpclb (or vice-versa).